<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>491</width>
    <height>420</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Andromeda Image Proccessing Software</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="gb_islemler">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>471</width>
      <height>381</height>
     </rect>
    </property>
    <property name="title">
     <string>Menu</string>
    </property>
    <widget class="QComboBox" name="cb_limits">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="currentText">
      <string>LOWER BOUND</string>
     </property>
     <item>
      <property name="text">
       <string>LOWER BOUND</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>UPPER BOUND</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="bt_images">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>451</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Choose Images To Process</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bt_file">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>451</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Choose Video To Process</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cb_showMask">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>170</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Show Mask</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bt_colorPicker">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>120</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>HSV Color Picker</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cb_maskCount">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>140</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="currentText">
      <string>1</string>
     </property>
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="lb_h_2">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>120</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Mask Count</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cb_currentMask">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>170</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="currentText">
      <string>1</string>
     </property>
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>5</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="lb_h_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Current Mask:</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>210</y>
       <width>471</width>
       <height>111</height>
      </rect>
     </property>
     <property name="title">
      <string>HSV Ranges</string>
     </property>
     <widget class="QLabel" name="lb_v">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>80</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>V</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>H</string>
      </property>
     </widget>
     <widget class="QLabel" name="lb_h">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>20</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>H</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>S</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>V</string>
      </property>
     </widget>
     <widget class="QSlider" name="slider_h">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>401</width>
        <height>22</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSlider" name="slider_v">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>401</width>
        <height>22</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="lb_s">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>50</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>S</string>
      </property>
     </widget>
     <widget class="QSlider" name="slider_s">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>401</width>
        <height>22</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="bt_saveMasks">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>330</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Save Current Masks</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cb_detection">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>190</y>
       <width>201</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Searching for Color</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bt_object">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>330</y>
       <width>281</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Object Detection</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
